### PR TITLE
Ibp 5405 remove dupe locations and update usage

### PR DESCRIPTION
**Pull request by clarysabel**

### CODE REVIEW

File: src/main/resources/liquibase/crop_changelog/22_1_0.xml [hunk 1]
```diff
@@ -39,5 +40,83 @@
...
+		  SET glocn = l1.locid
+		  WHERE g.glocn = l2.locid AND l1.locid &lt; l2.locid AND l1.lname = l2.lname 
+		  AND l1.ltype= l2.ltype AND l1.cntryid = l2.cntryid AND l1.locid=l1.snl1id;
+		
+		 UPDATE atributs a INNER JOIN location l1 INNER JOIN location l2
+		 SET a.alocn = l1.locid 
+		 WHERE a.alocn = l2.locid AND l1.locid &lt; l2.locid AND l1.lname = l2.lname 
+		 AND l1.ltype= l2.ltype AND l1.cntryid = l2.cntryid AND l1.locid=l1.snl1id;
+
+     UPDATE atributs_invalid_atype_bkp a INNER JOIN location l1 INNER JOIN location l2
```
Comment (clarysabel) [inline src/main/resources/liquibase/crop_changelog/22_1_0.xml:67]: @mchabito please remove updating atributs_invalid_atype_bkp, we are adding a changeset to drop this table as it is not further needed

File: src/main/resources/liquibase/crop_changelog/22_1_0.xml [hunk 2]
```diff
@@ -29,7 +30,7 @@
 			</sql>
 	</changeSet>
 
-	<changeSet author="cuenyad" id="v22.1.0-1">
+	<changeSet author="cuenyad" id="v22.1.0-2">
```
Comment (clarysabel) [inline src/main/resources/liquibase/crop_changelog/22_1_0.xml:33]: @mchabito revert changing the id

File: src/main/resources/liquibase/crop_changelog/20_3_0.xml [hunk 3]
```diff
@@ -1,9 +1,11 @@
 <?xml version="1.1" encoding="UTF-8" standalone="no"?>
+
```
Comment (clarysabel) [inline src/main/resources/liquibase/crop_changelog/20_3_0.xml:2]: @mchabito revert changes in this file

File: src/main/resources/liquibase/crop_changelog/22_1_0.xml [hunk 4]
```diff
@@ -39,5 +40,83 @@
...
+		 AND l1.locid &lt;l2.locid AND l1.lname = l2.lname 
+     AND l1.ltype= l2.ltype AND l1.cntryid = l2.cntryid AND l1.locid=l1.snl1id ;
+			
+     UPDATE  program_favorites f INNER JOIN location l1 INNER JOIN location l2  
+     SET f.entity_id= l1.locid 
+     WHERE  f.entity_id = l2.locid  AND l1.locid &lt; l2.locid
+     AND l1.lname = l2.lname AND l1.ltype= l2.ltype AND l1.cntryid = l2.cntryid 
+     AND l1.locid=l1.snl1id AND entity_type = "LOCATION";
+      
+     UPDATE atributs_aud g INNER JOIN location l1 INNER JOIN location l2
```
Comment (clarysabel) [inline src/main/resources/liquibase/crop_changelog/22_1_0.xml:100]: @mchabito I am checking what we should do with the aud tables.

Comment (clarysabel) [reply src/main/resources/liquibase/crop_changelog/22_1_0.xml:100]: Ok, with updating aud tables!

**Comment by clarysabel** [comment]
Hi @mchabito Please update this branch with master, we have added 2 new changesets

**Comment by mchabito** [comment]
@Rowena Tulod ***@***.***>
can you please do asap this change requested by Clarysabel?

Thanks

**Comment by mchabito** [comment]
done!

File: src/main/resources/liquibase/crop_changelog/20_3_0.xml (hunk 3 above)
Comment (clarysabel) [reply src/main/resources/liquibase/crop_changelog/20_3_0.xml:2]: @mchabito please revert this change, it was reverted but it was back in the last commit @rvtulod

**Comment by abatac** [comment]
Hi @rvtulod @mchabito The following columns must also be updated:

```
program_location_default.breeding_location_id
program_location_default.storage_location_id
```
@clarysabel

**Pull request merged**